roles/openshift_node/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 os_firewall_allow:
-- service: OpenShift kubelet
+- service: Kubernetes kubelet
```

Comment:
@aveshagarwal it's not uncommon to require user's to manage their own firewall configurations...  As long as we document it, I believe we will be okay.

I don't think we'd get docker or kubernetes to agree to let an outside service manage their firewall configurations, since they are being done dynamically according to changes in the system.